### ISSUE TITLE
3D File Viewer: Can't navigate to /res/gl in open dialog without manually typing location into location bar

### ISSUE THREAD

**Issue by tomuta** [SerenityOS/serenity#7297]
Steps to reproduce:

1. Open 3D File Viewer from the Serenity menu
2. Click File->Open
3. In the open dialog, click the root folder. Notice that there are no files or folders you can navigate to

```
3DFileViewer(49:49): Model was updated; dropping columns :(
[#0 3DFileViewer(49:49)]: Rejecting path '/' since it hasn't been unveiled with 'r' or 'b' permissions.
[#0 3DFileViewer(49:49)]: 0xc02edba1  Kernel::VFS::validate_path_against_process_veil(AK::StringView, int) +0x309
[#0 3DFileViewer(49:49)]: 0xc02f2266  Kernel::VFS::resolve_path(AK::StringView, Kernel::Custody&, AK::RefPtr<Kernel::Custody, AK::RefPtrTraits<Kernel::Custody> >*, int, int) +0x200
[#0 3DFileViewer(49:49)]: 0xc02f2f6b  Kernel::VFS::open(AK::StringView, int, unsigned short, Kernel::Custody&, AK::Optional<Kernel::UidAndGid>) +0x181
[#0 3DFileViewer(49:49)]: 0xc047293d  Kernel::Process::sys$open(AK::Userspace<Kernel::Syscall::SC_open_params const*>) +0x17d3
[#0 3DFileViewer(49:49)]: 0xc03eb152  syscall_handler +0x1e7d
[#0 3DFileViewer(49:49)]: 0xc03e929d  syscall_asm_entry +0x31
```

**Comment by linusg** [comment]
That's no accurate, you can type `/res` into the location bar and the files there will appear and can be opened. We're simply not unveiling `/` with `r` or `b` permissions, that's why it's empty.

https://github.com/SerenityOS/serenity/blob/832e9c6e02b3629db8d5303fb94ec9d0e7a7b145/Userland/Applications/3DFileViewer/main.cpp#L132-L145

**Comment by tomuta** [comment]
> That's no accurate, you can type `/res` into the location bar and the files there will appear and can be opened.

Well, I did not think of doing this... ;)

**Comment by linusg** [comment]
Regarding a fix, we might just want to drop the `unveil()`s and call it a day? It already `pledge()`s only `rpath` and not `wpath`, so that should be just fine. Even if we add more unveil paths and/or permissions, there's always going be some part that'll be impacted, short of not unveiling in the first place.

**Comment by linusg** [comment]
Fixed in df5f382b502fe18dd7762bd8ab830833bd22a5f6.
